pkg/controller/kogitoapp/kogitoapp_controller_test.go

```diff
@@ -191,21 +212,26 @@ func TestKogitoAppWithResource(t *testing.T) {
 		},
 	}
 
-	images := []runtime.Object{&isTag, &image1, &image2, &image3, &image4, &image5, &image6}
-	objs := []runtime.Object{kogitoapp}
+	return []runtime.Object{&isTag, &isTagBuild, &image1, &image2, &image3, &image4, &image5, &image6}
+}
+
+func setupSchema(kogitoapp *v1alpha1.KogitoApp) *runtime.Scheme {
 	// add to schemas to avoid: "failed to add object to fake client"
-	// Create a fake client to mock API calls.
 	s := meta.GetRegisteredSchema()
-	s.AddKnownTypes(v1alpha1.SchemeGroupVersion,
-		kogitoapp,
-		&v1alpha1.KogitoAppList{})
-	s.AddKnownTypes(appsv1.GroupVersion,
-		&appsv1.DeploymentConfig{},
-		&appsv1.DeploymentConfigList{})
+	s.AddKnownTypes(v1alpha1.SchemeGroupVersion, kogitoapp, &v1alpha1.KogitoAppList{})
+	s.AddKnownTypes(appsv1.GroupVersion, &appsv1.DeploymentConfig{}, &appsv1.DeploymentConfigList{})
 	s.AddKnownTypes(buildv1.GroupVersion, &buildv1.BuildConfig{}, &buildv1.BuildConfigList{})
 	s.AddKnownTypes(routev1.GroupVersion, &routev1.Route{}, &routev1.RouteList{})
 	s.AddKnownTypes(imgv1.GroupVersion, &imgv1.ImageStreamTag{}, &imgv1.ImageStream{}, &imgv1.ImageStreamList{})
 	s.AddKnownTypes(monv1.SchemeGroupVersion, &monv1.ServiceMonitor{}, &monv1.ServiceMonitorList{})
```

Comment:
Please add those types to the `meta.GetRegisteredSchema()` function instead.

Reply:
corrected